src/components/InputTodo.js

```
@@ -0,0 +1,53 @@
+/* eslint-disable react/no-access-state-in-setstate */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/state-in-constructor */
```

- [ ] Optional: Avoid disabling/ignoring errors.